scripts/setup-sdk.sh

```diff
@@ -32,3 +32,4 @@ sudo pip3 install -r bootloader/mcuboot/scripts/requirements.txt
 
 popd # "${NCS_ROOT}"
 
+cp -R "${this_dir}"/../boards/arm/nrf9160_avt9152 "${this_dir}"/../ncs/zephyr/boards/arm/nrf9160_avt9152/
```

Comment:
II would think that the board board need to go into "${NCS_ROOT}" not somewhere relative to the repo directory, so: ```cp -R "${this_dir}"/../boards/arm/nrf9160_avt9152 "${NCS_ROOT}"/zephyr/boards/arm/nrf9160_avt9152/ ``` ?

Also, I am not 100% sure about this, but I would think that the user can select the board in the IDE from an arbitrary directory location (like this_cloned_repo/boards/etc.). I'm pretty sure that's how I tested building your project. Maybe this copy is not even necessary as long as the automated build (if required) can refer to that board location and if the instructions in README explain how to do that. Basically point to different location in the readme instead of *for AVT9152-EVB: *<NCS_ROOT>/nrf/boards/arm/nrf9160_avt9152*

There is maybe also an additional problem when we create a project package maybe, where the user is supposed to be able to just download the project and not have to execute any scripts after downloading (I don't recall how the project needs to be set up after downloading.. it's been a while).

Reply:
it is not a must to have another copy of avt9152 board files inside ncs/zephyr/boards/arm.
it was created to allow user to build the project from command-line without providing the board files location.
i can change back to my earlier version without the folder copy if you see this unnecessary.

Reply:
have changed back to use the board files in /boards/arm/nrf9160_avt9152
have updated README.md to reflect the path as well